### PR TITLE
VTCCA-3728: DVR issue fix

### PR DESCRIPTION
**Pull request by darrell-plant**
Handling service call errors (use custom HttpReads over custom HttpErrorFunctions)

### CODE REVIEW

**Comment by PeterPerhac** [comment]
👍  LGTPP I think I can follow what's going on there. I was sorting out something very similar some time ago. It's a tough task

File: conf/logback.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
   <logger name="com.ning.http.client" level="${logger.com.ning.http.client:-INFO}"/>
 
-  <root level="DEBUG">
+  <root level="INFO">
```
Comment (habeeb01) [inline conf/logback.xml:16]: is this on purpose?

**Comment by habeeb01** [comment]
LGTM 👍  - apart from the comment re log level

File: conf/logback.xml (hunk 1 above)
Comment (darrell-plant) [reply conf/logback.xml:16]: yeah - I think I accidentally committed the DEBUG in my previous PR but that wasn't spotted - just resetting here.

**Pull request merged**